scripts/resource/README.md

```diff
@@ -0,0 +1,41 @@
+# Resource class generator
+
```

Comment:
maybe add that the script does not cover cases where args are mutually exclusive or optionally mandatory (i.e one or the other)?